<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMSClientClass</class>
 <widget class="QMainWindow" name="IMSClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMSClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTabWidget" name="tabWidget_main">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="tab_msgs">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_currentUser">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidget_msgData"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
               <item>
                <widget class="QLineEdit" name="lineEdit_sendData"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_sendMsg">
                 <property name="text">
                  <string>发送</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_persons">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>联系人</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidget_userData"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_message">
          <property name="text">
           <string>消息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_persons">
          <property name="text">
           <string>联系人</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IMSClient.qrc"/>
 </resources>
 <connections/>
</ui>
